.github/workflows/gh-pages.yml

```diff
@@ -22,11 +22,16 @@ concurrency:
 jobs:
   # Build job
   build:
-    runs-on: ubuntu-24.04
+    runs-on: ubuntu-latest
```

Comment:
I'd rather have us locked in on a known version, like we do with rpm. So rather than change this to -latest, change rpm-web to whatever the version we want just now.

I hate the kind of surprises where s*** breaks without notice. Maybe the risk of breakage isn't that bad with the web stuff as it is for rpm itself but still...

Reply:
Ah good point, it's not like this never happened, quite the contrary :sweat_smile: Will fix (in rpm-web too).